### DISCUSSION TITLE
HTTPS Issues with Docker

### DISCUSSION THREAD

**Discussion by bit0fun** [Leantime/leantime#962] [Q&A] [answered]
I’ve been having a few issues with getting leantime setup with https with docker, even though http seems to work just fine.

For the setup, I’ve used the docker-leantime repo for the compose file, I have an nginx as a proxy to route traffic with certificate files setup there for correct port forwarding. The docker instance is also listening on port 443 and port 80, though it seems to really only like port 80. I’ve tried setting port 80 to be used as the port forward from the external SSL port, though that did not seem to work. I attempted to map the certificate files from the host to the docker container, but can’t check if they are seen properly. 

Switching the LEAN_APP_URL from https to http will make everything work properly, though of course this isn’t ideal.

Not entirely sure what to do with it, feel like I’m just missing something. 

Feel free to ask for more info if needed

**Comment by marcelfolaron** [comment]
I think the majority of users are doing what you described:
Set up nginx and route ssl traffic to that, nginx proxies and routes
traffic via port 80 to leantime.
This shouldn't be a problem if the traffic is contained within the same
network.

You can join our discord channel to talk to some of the folks running this
set up. We do have a few who have done the complete ssl flow.

**Reply by bit0fun** [reply]
Thanks! Will do. Appreciate the quick response

**Comment by DAVIDE-ph** [comment]
I have the same problem too. I installed leantime on docker using image `"leantime/leantime:latest" -p 1780:80` because port 80 is used by NginxProxyManager, then using ng I created a subdomain with ssl certificate pointing to serverip:1780. page opens but there are certain Errors like pop-up windows fail.
Does anyone have any solution without using port 80?

**Comment by marcelfolaron** [comment, marked as answer]
Do you have LEAN_APP_URL set?

**Reply by DAVIDE-ph** [reply]
I hadn't defined before. Now it works properly thanks for the support


![Untitled-1](https://github.com/Leantime/leantime/assets/97435527/8dfaa285-46e9-429a-bff4-be25dfeacb95)

Thank you again

**Reply by xhk416x** [reply]
This should be somewhere in the docker setup as a note.
E.g.
**NOTE: If using a proxy like NPM, Caddy, or Traefik, $LEAN_APP_URL must be set for the website to be displayed properly.

**Comment by marcelfolaron** [comment]
Great, glad it was an easy fix
